CHANGELOG.md

```diff
@@ -2,16 +2,18 @@
 
 ### Enhancements
 * <New feature description> (PR [#????](https://github.com/realm/realm-core/pull/????))
-* None.
+* Normalize sync connection parameters for device information ([#6029](https://github.com/realm/realm-core/issues/6029))
```

Comment:
I don't think we need a separate enhancement line for this. It's not really a public-facing enhancement, just a change in what metadata we track.